docs/DCO.md

```diff
@@ -0,0 +1,21 @@
+Developer Certificate of Origin
```

Comment:
```suggestion
[Developer Certificate of Origin](https://developercertificate.org/)
```

Reply:
@blueQcloud did you have an opinion on this one?

Reply:
hmm, you said you wanted to be consistent with the links?